packages/core/src/fhir-to-cda/cda-templates/author.ts

```diff
@@ -0,0 +1,23 @@
+import { Organization } from "@medplum/fhirtypes";
+import {
+  buildAddress,
+  buildRepresentedOrganization,
+  buildTelecom,
+  withNullFlavor,
+} from "./commons";
+import { CDAAuthor } from "../cda-types/shared-types";
+import { rootAttribute, valueAttribute } from "./constants";
+
+export function buildAuthor(organization: Organization): CDAAuthor {
+  const author = {
+    time: withNullFlavor(undefined, valueAttribute),
+    assignedAuthor: {
+      id: withNullFlavor(organization.id, rootAttribute),
+      addr: buildAddress(organization.address),
+      telecom: buildTelecom(organization.telecom),
+      representedOrganization: buildRepresentedOrganization(organization), // TODO: organization shouldn't actually be required
+      // TODO: one of (assignedPerson / assignedAuthoringDevice) should be required
```

Comment:
Removing these comments as per the discussion - [context](https://metriport.slack.com/archives/C04DMKE9DME/p1713459442766429?thread_ts=1713448810.552229&cid=C04DMKE9DME)

Reply:
But no, we aren't really keeping track on a ticket - since this whole converter is in a very active development stage, where things are changing at a rapid pace, and most of them will be resolved soon

Reply:
I mean, we might address those or not as we go about coding. If its an important TODO, I'd suggest we have it on a ticket. If it's more like a "nice to have" kinda thing, then for sure no need to worry too much.

It's just that, as a reviewer, this is the only opportunity I have for sure to make sure we're not missing something important, since we'll only have it on a PR again if someone remembers to address it.